cg/meta/workflow/balsamic.py

```diff
@@ -328,18 +326,12 @@ def get_verified_samples(
             sample_data.get(normal_sample_id).get("concatenated_path") if normal_sample_id else None
         )
         if normal_sample_id and not tumor_sample_id:
-            if force_normal:
-                LOG.warning(
-                    f"Only a normal sample was found for case {case_id}. "
-                    f"Balsamic analysis will treat it as a tumor sample."
-                )
-                tumor_sample_id, tumor_sample_path = normal_sample_id, normal_sample_path
-                normal_sample_id, normal_sample_path = None, None
-            else:
-                LOG.error(
-                    f"Case {case_id} only has a normal sample. Use the --force-normal flag to treat it as a tumor."
-                )
-                raise BalsamicStartError
+            LOG.warning(
+                f"Only a normal sample was found for case {case_id}. "
+                f"Balsamic analysis will treat it as a tumor sample."
+            )
+            tumor_sample_id, tumor_sample_path = normal_sample_id, normal_sample_path
+            normal_sample_id, normal_sample_path = None, None
 
         return {
```

Comment:
maybe for the future, but it would also be nice to have this dict in a model, same goes for the sample_data-

Reply:
I opened an issue some time ago to refactor this part, but haven't found time yet: https://github.com/Clinical-Genomics/cg/issues/1724